### DISCUSSION TITLE
Schrödinger's Release: A public release is not the list, except for some people, but in the wrong order

### DISCUSSION THREAD

**Discussion by DropSnorz** [community/community#76962] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hello,

In my repository [OwlPlug](https://github.com/DropSnorz/OwlPlug), I've released versions [1.24.0](https://github.com/DropSnorz/OwlPlug/releases/tag/1.24.0) on Aug. 18 and [1.25.0](https://github.com/DropSnorz/OwlPlug/releases/tag/1.25.0) today. Both are directly accessible by URL.

When I'm on the [Release List](https://github.com/DropSnorz/OwlPlug/releases) as anonymous (logged out), I can't see the release 1.24.0

![image](https://github.com/community/community/assets/15964929/6aa70b2f-1cdb-459a-9681-191267afd652)

But when I'm logged in with my account (@Dropsnorz), I can see the release 1.24.0 in the list, but on top of 1.25.0. The order is wrong, the 1.25.0 is more recent and flagged as the latest, so it should appear on top.

![image](https://github.com/community/community/assets/15964929/93d4e232-9605-4847-a69e-8ecb55bd4e00)

It seems something is not working correctly here. The release 1.24.0 is not in Draft, and I can't see it using the REST API.
Can you see the Release 1.24.0 in the list? Do you have an idea of what's happening here?

Thanks,

**Comment by Raicuparta** [comment, marked as answer]
I've seen this before. At the time, it was fixed by just editing the release and saving it without changing anything.

**Reply by DropSnorz** [reply]
After updating the release it's now fully visible. Thanks 🙂
